src/utils/isAuthorizedDelegatedAccount.ts

```diff
@@ -0,0 +1,29 @@
+import { Address } from '$/types'
+import handleError from '$/utils/handleError'
+import { Contract, providers } from 'ethers'
+import * as DelegatedAccessRegistry from '../contracts/DelegatedAccessRegistry.sol/DelegatedAccessRegistry.json'
+const DelegatedAccessRegistryAddress = '0xf5803cdA6352c515Ee11256EAA547BE8422cC4EE'
+
+export default async function isAuthorizedDelegatedAccount(
```

Comment:
`isAuthorizedDelegatedAccount` should always return a boolean (`is*(): boolean`). Currently it's `undefined | boolean`.